packages/native-stack/package.json

```diff
@@ -61,7 +61,7 @@
     "react": "*",
     "react-native": "*",
     "react-native-safe-area-context": ">= 3.0.0",
-    "react-native-screens": ">= 3.0.0"
+    "react-native-screens": ">= 3.19.0"
```

Comment:
I set this to a fictitious future version, because it relies on a change to react-native-screens

Reply:
@jaredly the peer dependency shouldn't be bumped, the package is still compatible with react native screens 3, only the new feature needs a newer version so that can be documented separately